### PR TITLE
Add missing policies to PolicyDefinition

### PR DESCRIPTION
**Pull request by inikulshin**
Based on
image

### CODE REVIEW

**Review by zidad — APPROVED** [review]

**Comment by zidad** [comment]
Wow, many thanks!

**Comment by inikulshin** [comment]
> Wow, many thanks!

However, the PR breaks public API.
I'll make JsonStringEnumConverterEx internal, but all other changes are mandatory.

**Comment by inikulshin** [comment]
Now it fails because of the last EOL symbol in [EasyNetQ.Management.Client.approved.txt](https://github.com/EasyNetQ/EasyNetQ.Management.Client/blob/master/Source/EasyNetQ.Management.Client.ApprovalTests/EasyNetQ.Management.Client.approved.txt).
But I can't fix it from the browser, like all previous commits I did 😠

**Review by zidad — APPROVED** [review]

**Comment by inikulshin** [comment]
Hallelujah!

I'll make some more PRs (at least 1) soon.
Will it be possible to create a new tag and publish new [NuGet package](https://www.nuget.org/packages/EasyNetQ.Management.Client#versions-body-tab) soon?

**Review by Pliner — APPROVED** [review]
Thanks!

**Comment by Pliner** [comment]
> Hallelujah!
> 
> I'll make some more PRs (at least 1) soon. Will it be possible to create a new tag and publish new [NuGet package](https://www.nuget.org/packages/EasyNetQ.Management.Client#versions-body-tab) soon?

Sure. We will be able to release a new dev version available on NuGet.

**Comment by Pliner** [comment]
https://www.nuget.org/packages/EasyNetQ.Management.Client/3.0.0-alpha3

File: Source/EasyNetQ.Management.Client/Serialization/JsonStringEnumConverterEx.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+using System.Text.Json;
+
+namespace EasyNetQ.Management.Client.Serialization;
+
+// https://github.com/dotnet/runtime/issues/31081#issuecomment-848697673
+internal class JsonStringEnumConverterEx<TEnum> : JsonConverter<TEnum> where TEnum : struct, Enum
+{
+    private readonly Dictionary<TEnum, string> _enumToString = new Dictionary<TEnum, string>();
```
Comment (sungam3r) [inline Source/EasyNetQ.Management.Client/Serialization/JsonStringEnumConverterEx.cs:10]: All fields and constructor can be made static.

**Pull request merged**